File: static_cdn/media_root/submissions/Deepak123/WINP.cpp
#include<iostream>
using namespace std;
int sizeoof(string s);
int main()
{
	int t,i,k,c1=0,c2=0;
	string s;
	cin>>t;
	for(int i=0;i<t;i++)
	{
		cin>>s;
		k=sizeoof(s);
		if(k==51)
		{
			c1++;
		}
		if(k==73)
		{
			c2++;
		}
	}
	if(c1>c2)
	{
		cout<<"dheeraj";
	}
	if(c2>c1)
	{
		cout<<"ranjeet";
	}
}
int sizeoof(string s)
{
int	l=s.length();
int i,k=0;
	for(i=0;i<l;i++)
	{
	k=k+s[i];	
	}
	k=k-60*l;
	return k;
}

File: static_cdn/media_root/submissions/rabada/WINP.py
d = {}
for i in range(ord('a'),ord('z')+1):
    d[chr(i)] = i-ord('a')+1

a=73
b=51
a1=0
b1=0
for i in range(0,int(input())):
    c=0
    e = str(input())
    for i in range(0,len(e)):
        c = c + d[e[i]]
    if(c == a):
        a1 = a1 + 1
    elif (c == b):
        b1 = b1 + 1

if (a1 > b1):
    print('ranjeet')
else:
    print('dheeraj')
